decidim-proposals/app/helpers/decidim/proposals/application_helper.rb

```diff
@@ -138,18 +138,16 @@ def votes_given
         ).count
       end
 
+      # deprecated
       def votes_count_for(model, from_proposals_list)
         render partial: "decidim/proposals/proposals/participatory_texts/proposal_votes_count.html", locals: { proposal: model, from_proposals_list: }
       end
 
+      # deprecated
```

Comment:
```suggestion
      # REDESIGN: deprecated
```

Or maybe @entantoencuanto could we remove this? Since it's in proposals module no other module is using it

Reply:
Removed